docs/specs/oidc-sso-device.md

```diff
@@ -0,0 +1,84 @@
+# Device Single Sign On
+
+This is a SSO mechanism to share authentication session between applications on a single device without using browser cookies.
+
+## Abstract
+
+This is a implementation based on the [OIDC Native SSO Specification](https://openid.net/specs/openid-connect-native-sso-1_0.html).
+
+It is expected to be used by mobile apps to share the authentication session with another app installed in the same device.
+
+The authgear server will generate a device token with an id token for a single device. Applications on the device are able to share authentication session using that device token and id token.
+
+Applications which are able to share authentication session are expected to be signed by the same vendor certificate, and are expected share the device token and id token by a shared storage which only accessible by applications signed by the same vendor certificate.
+
+## Implementation details
+
+### Server-side
+
+- Token Endpoint
+
+  - Support a new scope `device_sso`
```

Comment:
Add new a paragraph to specify that when `device_sso` is included in scope, then `x_suppress_idp_session_cookie` is implied as true.

Also specify that `x_sso_enabled` CANNOT be used together with `device_sso`, doing so will result in OAuth error in the authorization endpoint.